docs/Growing.md

```diff
@@ -415,12 +415,119 @@ with `ctx.redirectTo()`, you can use the flash, accessible through `ctx.flash`.
 ```js
 const flash = await ctx.flash();
 flash.message = 'Everything is fine.';
-return ctx.redirectTo('goodbye');
+await ctx.redirectTo('goodbye');
 ```
 
 Just remember that all session data gets serialized to JSON and stored in encrypted cookies, which usually have a `4096`
 byte (4KiB) limit, depending on browser.
 
+## Final prototype
+
+A final `myapp.js` prototype passing all of the tests above could look like this.
+
+```js
+import mojo from '@mojojs/core';
+import Users from './models/users.js';
+
+// Set cookie secret
```

Comment:
The original says `# Make signed cookies tamper resistant`. Which sounds much better.

Reply:
Note that cookies are encrypted here though.

Reply:
I know, that's why I changed it from tamper resistant, which isn't really true anymore once they are encrypted.

Reply:
`Make encrypted cookies more secure with custom secrets` maybe?